package/network/services/hostapd/files/hostapd.sh

```diff
@@ -640,7 +640,7 @@ hostapd_set_bss_options() {
 		;;
 	esac
 
-	local auth_algs=$((($auth_mode_shared << 1) | $auth_mode_open))
+	local auth_algs="$((($auth_mode_shared << 1) | $auth_mode_open))"
```

Comment:
Won't $(()) auto-quote?

Reply:
you're right! i didn't know (".. treated as if it were within double quotes")

this change is supposed to address a problem with syntax highlighting - not with the shell.
i regularly add quotes at this location because if i scroll by from the wrong angle many of the following lines are highlighted as if the parenthesis wasn't closed (purple in vim).

Reply:
this happens to me regularly:
![vim](https://user-images.githubusercontent.com/5165324/104327296-e66e4980-54ea-11eb-841b-c69a75f38cd9.png)

not really an openwrt-issue - especially with your comment that it's technically unnecessary to quote there.